libp2p/upgrademngrs/muxedupgrade.nim

```diff
@@ -59,6 +59,8 @@ proc mux*(u: MuxedUpgrade, conn: Connection): Future[Muxer] {.async, gcsafe.} =
 
   try:
     await u.identify(muxer)
+  except CancelledError:
```

Comment:
This causes issues with `raises` because the type of the exception is lost on `raise`.